misago/users/setupnewuser.py

```diff
@@ -27,3 +27,5 @@ def set_default_subscription_options(settings, user):
 
     replied_threads = SUBSCRIPTION_CHOICES[settings.subscribe_reply]
     user.subscribe_to_replied_threads = replied_threads
+
+    user.save(update_fields=['subscribe_to_replied_threads'])
```

Comment:
This list is missing `subscribe_to_started_threads`.  Also please use double quote for commenting strings:
```suggestion
    user.save(update_fields=[
        "subscribe_to_started_threads", "subscribe_to_replied_threads"
    ])
```

Reply:
@rafalp 
Thank you for the correction.